pkgs/top-level/all-packages.nix

```diff
@@ -32709,6 +32709,8 @@ with pkgs;
     wlroots = wlroots_0_15;
   };
 
+  waybar-mpris = callPackage ../applications/misc/waybar/modules/waybar-mpris { };
```

Comment:
Please move the package to `pkgs/by-name/wa/waybar-mpris/package.nix` and remove this line, thanks.

Reply:
Sorry, didn't know about that other directory structure, I've always used top-level. I'll update it.